### PR TITLE
fix: advanced-setup Gradle code snippets for RN

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: src/platforms/react-native/advanced-setup.mdx [hunk 1]
```diff
@@ -86,21 +86,21 @@ For Android, we hook into Gradle for the source map build process. When you run
 
 We enable the Gradle integration in your `android/app/build.gradle` file by adding the following line after the `react.gradle` one:
 
-```java
+```groovy
```
Comment (bruno-garcia) [inline src/platforms/react-native/advanced-setup.mdx:89]:
```suggestion
```groovy {filename:android/app/build.gradle}
```

File: src/platforms/react-native/advanced-setup.mdx [hunk 2]
```diff
@@ -86,21 +86,21 @@ For Android, we hook into Gradle for the source map build process. When you run
...
 apply from: "../../node_modules/@sentry/react-native/sentry.gradle"
 ```
 
 You can also enable logging for `sentry-cli` by adding this config before the above `apply from:` line:
 
-```java
+```groovy
```
Comment (bruno-garcia) [inline src/platforms/react-native/advanced-setup.mdx:95]:
```suggestion
```groovy {filename:android/app/build.gradle}
```

File: src/platforms/react-native/advanced-setup.mdx [hunk 3]
```diff
@@ -86,21 +86,21 @@ For Android, we hook into Gradle for the source map build process. When you run
...
 project.ext.sentryCli = [
     logLevel: "debug"
 ]
 ```
 
 We also support fetching different `sentry.properties` files for different flavors. For that, you need to add:
 
-```java
+```groovy
```
Comment (bruno-garcia) [inline src/platforms/react-native/advanced-setup.mdx:103]:
```suggestion
```groovy {filename:android/app/build.gradle}
```

File: src/platforms/react-native/advanced-setup.mdx [hunk 4]
```diff
@@ -113,7 +113,7 @@ We recommend leaving `logLevel: "debug"` since we look for specific `sentry.prop
 
 Include the project by adding it to your dependency list in `app/build.gradle`:
 
-```java
+```groovy
```
Comment (bruno-garcia) [inline src/platforms/react-native/advanced-setup.mdx:116]:
```suggestion
```groovy {filename:android/app/build.gradle}
```

File: src/platforms/react-native/advanced-setup.mdx [hunk 5]
```diff
@@ -113,7 +113,7 @@ We recommend leaving `logLevel: "debug"` since we look for specific `sentry.prop
 
 Include the project by adding it to your dependency list in `app/build.gradle`:
```
Comment (bruno-garcia) [inline src/platforms/react-native/advanced-setup.mdx:114]: Ins't this `android/..` ?

```suggestion
Include the project by adding it to your dependency list in `android/app/build.gradle`:
```

File: src/platforms/react-native/advanced-setup.mdx [hunk 6]
```diff
@@ -140,7 +140,7 @@ public class MainApplication extends Application implements ReactApplication {
 
 Add the following to your `settings.gradle` file:
 
-```java
+```groovy
```
Comment (bruno-garcia) [inline src/platforms/react-native/advanced-setup.mdx:143]:
```suggestion
```groovy {filename:android/app/build.gradle}
```

**Comment by marandaneto** [comment]
@jennmueng could you confirm those file dirs? if they are `android/app/build.gradle` or `app/build.gradle` or even the `gradle` file on the node folder? just to be sure we have the correct dirs, I don't remember by heart.

**Comment by jennmueng** [comment]
@marandaneto Should be correct. There’s `android/build.gradle` for the project and `android/app/build.gradle` for the app.

**Comment by marandaneto** [comment]
> @marandaneto Should be correct. There’s `android/build.gradle` for the project and `android/app/build.gradle` for the app.

I know there are 2 of them.
I just don't know if all the mentions to those files on the advanced setup page are correct, @bruno-garcia is suggesting a change for example https://github.com/getsentry/sentry-docs/pull/2638#discussion_r515676703 and I am not sure if its right or not, so ideally we would check up all the gradle files mentioned on this page, to be sure it states the right file/folder

File: src/platforms/react-native/advanced-setup.mdx (hunk 6 above)
Comment (jennmueng) [reply src/platforms/react-native/advanced-setup.mdx:143]: Looks like this should be `settings.gradle` and not `app/build.gradle`
```suggestion
```groovy {filename:android/settings.gradle}
```

**Comment by jennmueng** [comment]
> > @marandaneto Should be correct. There’s `android/build.gradle` for the project and `android/app/build.gradle` for the app.
> 
> I know there are 2 of them.
> I just don't know if all the mentions to those files on the advanced setup page are correct, @bruno-garcia is suggesting a change for example [#2638 (comment)](https://github.com/getsentry/sentry-docs/pull/2638#discussion_r515676703) and I am not sure if its right or not, so ideally we would check up all the gradle files mentioned on this page, to be sure it states the right file/folder

Oh my bad, I see now. Apart from the `settings.gradle` one that I just replied to, they should all be for `android/app/build.gradle`.

**Pull request merged**